etc/ci/performance/test_all.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+set -o errexit
+set -o nounset
+set -o pipefail
+
+case "$1" in
+  --servo)
+    engine=""
+    ;;
+  --gecko)
+    engine="--engine gecko"
+    ;;
+  *)
+    echo "Can't understand the argument, using the default servo engine"
```

Comment:
Instead of defaulting to `servo` in this case, we should just error out.
It would be nice to handle `--help` as well and print usage, but that's not necessary.